webservice/routes/api.php

```diff
@@ -3,11 +3,13 @@
 Route::group([
     'middleware' => ['cors', 'api'],
 ], function () {
-    Route::post('/login', 'LoginController@login');
+    Route::post('/auth/issue', 'AuthController@issueToken');
```

Comment:
Suggestions are always welcome! =D
Accepted suggestion.

Following this line of thought, the method to revoke and update the token will also be changed, to:

Auth/token/revoke
Auth/token/refresh

OK?